grype/matcher/apk/matcher.go

```diff
@@ -26,22 +29,19 @@ func (m *Matcher) Type() match.MatcherType {
 func (m *Matcher) Match(store vulnerability.Provider, d *distro.Distro, p pkg.Package) ([]match.Match, error) {
 	var matches = make([]match.Match, 0)
 
-	// find Alpine SecDB matches for the given package name and version
-	secDbMatches, err := common.FindMatchesByPackageDistro(store, d, p, m.Type())
+	//direct matches with package
```

Comment:
(repeated for all similar cases) For readability, it's nice to have a space between the `//` and the comment text. I see this is already done in some places, and I'd recommend making all comments like this